### ISSUE TITLE
Cyclic Inventory Upgrade disappears when leaving the End through the Dragon portal

### ISSUE THREAD

**Issue by OriginalDahc** [NillerMedDild/Enigmatica2Expert#725]
**Issue Description**
When leaving the End through the portal generated after killing the End Dragon, the Inventory Upgrade obtained by eating the Inventory Upgrade cake locks again, requiring another cake to unlock.

**Modpack version (Do not use "latest")**
Version 1.54

**Logs**
N/A

**Screenshots**
N/A

**Additional context**
I also noticed my hearts dropped to a single full bar. Might be related, but also suggests the underlying engine just does this.

**Comment by spdkils** [comment]
I tracked it down to the cyclic config... After about 2 hours.... I removed every mod by cutting it in 1/2 over and over, even with NO other mods, it still does it.

Then I deleted the cylic config..... had a generate a new one.

Works fine.

. . . .

The problem is possibly that inventory is disabled, and it takes the others with it, testing now.

**Comment by spdkils** [comment]
Yeah, left all mods in, just regenerated config, and works fine... Just need to re-disable what you disabled, and see if it was just some syntax error hidden in here since the files are re-organized a lot.

**Comment by spdkils** [comment]
It's a Cylic mod bug, reported over there...https://github.com/Lothrazar/Cyclic/issues/1069
